src/inmanta/data/__init__.py

```diff
@@ -4875,7 +4869,12 @@ async def copy_resources_from_unchanged_resource_set(
                         ELSE 'available'::resourcestate
                         END
                     ) AS status,
-                    jsonb_set(r.attributes, '{{"version"}}'::text[], to_jsonb($3::integer)) AS attributes,
+                    (
+                        CASE WHEN r.attributes ? 'version'
+                        THEN jsonb_set(r.attributes, '{{"version"}}'::text[], to_jsonb($3::integer))
+                        ELSE r.attributes
+                        END
+                    ) AS attributes,
```

Comment:
Why not just delete it in a schema migration script? Or at least delete it here instead of updating it?

Reply:
Good point. I completely overlooked that. I think the removing the version field using a database migration script is the way to go. I will fix this.